src/Codeception/Test/Filter.php

```diff
@@ -1,70 +1,56 @@
 <?php
 
-declare(strict_types=1);
+namespace Codeception\Test;
```

Comment:
please keep strict types in this class.

Reply:
This is actually a new class, it just contains some code from the old file :)

Reply:
Fixed, anything else?

Reply:
that's all
Thank you